github/issues.go

```diff
@@ -36,6 +36,7 @@ type Issue struct {
 	Milestone        *Milestone        `json:"milestone,omitempty"`
 	PullRequestLinks *PullRequestLinks `json:"pull_request,omitempty"`
 	Repository       *Repository       `json:"repository,omitempty"`
+	Reactions        *ReactionsSummary `json:"reactions,omitempty"`
```

Comment:
> I didn't want any confusion with `type Reaction struct {...}`.

I was worried there was a stronger reason, like a naming conflict because `Reactions` was already taken by something else.

If that's not it, I think `Reactions` is much nicer and more consistent.

IMO `Reaction` and `Reactions` are fine to have, and they are great names/description for the types. We also have https://godoc.org/github.com/google/go-github/github#Page and https://godoc.org/github.com/google/go-github/github#Pages.